pkg/trie/triedb/codec/node.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2024 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package codec
+
+const ChildrenCapacity = 16
+
+// MerkleValue is a helper enum to differentiate between inline and hashed nodes
+// https://spec.polkadot.network/chap-state#defn-merkle-value
+type MerkleValue interface {
+	_isMerkleValue()
```

Comment:
why the _ syntax? Im familiar with prefixing unexported global variables with _ as seen [here](https://github.com/uber-go/guide/blob/master/style.md#prefix-unexported-globals-with-_), does this apply for functions as well?

Reply:
I personally dont love the syntax if it's not needed

Reply:
Taken from [here](https://zackoverflow.dev/writing/hacking-go-to-give-it-sumtypes#attempt-2-interfaces) but using single underscore instead of double

Reply:
Changed! thks!